src/renderer/wallet/main/toolbar.jsx

```diff
@@ -167,10 +183,10 @@ const ProfileList = withStyles(profileStyle)(
 									)}
 								</Grid>
 								<Grid item sm={8} className={classes.profileName}>
-									<Typography variant="h6">
+									<Typography variant="subtitle1" style={{ marginBottom: '5px' }}>
```

Comment:
It's better to try and avoid inline styles as much as possible.
I would suggest moving this to the profileName css rules.